test/integration/short/client-each-row-tests.js

```diff
@@ -125,10 +125,6 @@ describe('Client', function () {
             assert.ifError(err);
             assert.strictEqual(rowCount, 100);
             assert.strictEqual(rowCount, result.rowLength);
-            assert.ok(result.rowLengthArray);
```

Comment:
`rowLengthArray` was never part of the public API, it was meant for testing but it's not a good way to expose just to test it, so I took the chance to remove it.